src/USER-ADIOS/dump_atom_adios.cpp

```diff
@@ -0,0 +1,314 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Norbert Podhorszki (ORNL)
+------------------------------------------------------------------------- */
+
+#include <string.h>
```

Comment:
please use c++ style includes, i.e. `#include <cstring>` instead of `#include <string.h>`